web/tasks/Wiki-Auto.md

```diff
@@ -1,119 +1,213 @@
+    'She is a right handed bat.'],
+   'simple_sentence_id': ['simple-702227-0-0', 'simple-702227-0-1']},
+  'simple_article_id': 702227,
+  'simple_article_title': 'Lata Mondal',
+  'simple_article_url': 'https://simple.wikipedia.org/wiki?curid=702227'}}
+```
+
+Finally, the `auto_acl` config was obtained by selecting the aligned pairs of sentences from `auto` to provide a ready-to-go aligned dataset to train a sequence-to-sequence system, so an instance is a single pair of sentences:
+```
+{'normal_sentence': 'In early work, Rutherford discovered the concept of radioactive half-life , the radioactive element radon, and differentiated and named alpha and beta radiation .\n',
+ 'simple_sentence': 'Rutherford discovered the radioactive half-life, and the three parts of radiation which he named Alpha, Beta, and Gamma.\n'}
+```
+
+Thus, for training a text simplification model for GEM, the data with the `auto_acl` config can be directly used.
 
 ### Data Fields
 
-[More Information Needed]
+The data has the following field:
+- `normal_sentence`: a sentence from English Wikipedia.
+- `normal_sentence_id`: a unique ID for each English Wikipedia sentence. The last two dash-separated numbers correspond to the paragraph number in the article and the sentence number in the paragraph.
+- `simple_sentence`: a sentence from Simple English Wikipedia.
+- `simple_sentence_id`: a unique ID for each Simple English Wikipedia sentence. The last two dash-separated numbers correspond to the paragraph number in the article and the sentence number in the paragraph.
+- `alignment_label`: signifies whether a pair of sentences is aligned: labels are `>=1:aligned` and `0:notAligned`
+- `paragraph_alignment`: a first step of alignment mapping English and Simple English paragraphs from linked articles
+- `sentence_alignment`: the full alignment mapping English and Simple English sentences from linked articles
 
 ### Data Statistics
 
-[More Information Needed]
+In `auto`, the `part_2` split corresponds to the articles used in `manual`, and `part_1` has the rest of Wikipedia.
+
+The `manual` config is provided with a `train`/`dev`/`test` split with the following amounts of data:
+|                            | Tain   | Dev   | Test   |
+| -----                      | ------ | ----- | ----   |
+| Total sentence pairs       | 373801 | 73249 | 118074 |
+| Aligned sentence pairs     |  1889  |  346  | 677    |
+
+The `auto_acl` has 488,332 complex-sentence pairs that are to be used for training the model. The average sentence length for complex and simple sentences are 26.6 and 18.7 respectively.
 
 ## Dataset Creation
 
 ### Curation Rationale
 
-[More Information Needed]
+Wiki-Auto provides a new version of the Wikipedia corpus that is larger, contains 75% less defective pairs and has more complex rewrites than the previous WIKILARGE dataset.  
 
 ### Communicative Goal
 
-[More Information Needed]
```

Comment:
Can you add something here? It'd be sufficient to write something like "The goal is to communicate the same information as the source sentence in an easier-to-understand manner" or so.